src/main/java/de/themorpheus/edu/taskservice/controller/TaskController.java

```diff
@@ -101,12 +103,16 @@
 		return ControllerResult.of(this.taskRepository.findAll());
 	}
 
-	public ControllerResult<List<TaskModel>> getTasksFromLecture(String lectureNameKey) {
+	public ControllerResult<List<TaskModel>> getTasksFromLecture(String lectureNameKey, boolean showBanned) {
 		ControllerResult<LectureModel> lectureResult = this.lectureController
 			.getLectureByNameKey(lectureNameKey);
 		if (lectureResult.isResultNotPresent()) return ControllerResult.of(Error.NOT_FOUND, Constants.Lecture.NAME_KEY);
```

Comment:
Use ControllerResult.ret

Reply:
Why in this line?
Did you mean in the next line?

Reply:
no use ControllerResult.ret(lectureResult)